src/AccessibilityTranslator.ts

```diff
@@ -1,34 +1,40 @@
 import _ from 'lodash';
-import React, { ReactElement } from 'react';
-import PropTypes from 'prop-types';
+import { breadth } from 'treeverse';
 import { ROLES } from './Roles';
 import { getTagNameForDisplayObject } from './utils/roleUtils';
+import { createElement, updateElement } from './utils/domUtils';
 import { AccessibleDisplayObject } from './RoleObjects/AccessibilityObject';
 
-interface Props {
-  stage: object;
-}
-
 type ElementBounds = {
   height?: number;
   width?: number;
   x: number;
   y: number;
 };
 
-type DisplayObjectReactProps = {
+type EventHandler = (evt: Event) => void;
+
+export type DisplayObjectReactProps = {
```

Comment:
- I would strongly consider renaming all "react" references everywhere except the API, and put in a deprecation warning for `_reactProps` and `reactProps`

Reply:
Added deprecation warning for `reactProps` in proxy handler So, every time reactProps is accessed a warning is posted in the console.

Reply:
Removed deprecation warning!